lib/debug/frame_filter.rb

```diff
@@ -0,0 +1,10 @@
+module DEBUGGER__
```

Comment:
This could be required alone so extensions don't need to require the debugger and activate the debugger unnecessarily.